src/hotspot/share/classfile/classLoaderData.hpp

```diff
@@ -327,6 +327,10 @@ class ClassLoaderData : public CHeapObj<mtClass> {
   const char* loader_name_and_id() const;
   Symbol* name_and_id() const { return _name_and_id; }
 
+  unsigned identity_hash() const {
+    return (unsigned)((uintptr_t)this >> (LogMinObjAlignmentInBytes + 3));
```

Comment:
LogMinObjAlignmentInBytes describes the alignment requirements for Java objects. I know that some other places in the VM reuse this for generating hash codes, but I don't think that's appropriate, and they should be cleaned out.

Reply:
Hi,
I think it makes sense.
How about filing a new JBS issue to fix the problem you mentioned or you think we should fix it in this patch?

Reply:
I created a CR for this:
https://bugs.openjdk.java.net/browse/JDK-8267303

but I still think that this usage should not be introduced in this patch.

Reply:
okay, removed.

Reply:
This seems fine to me.

Reply:
@stefank 
Could you review it again?

Thanks,
Denghui